check.sh

```diff
@@ -0,0 +1,7 @@
+set -e
+
+black .
+find * -name "*.py" -print | xargs flake8
+pytest -vv
+
+echo done
```

Comment:
I'd prefer to do this with pre-commit rather than a shell script, I think there's a config file for it already in the repo, we can talk about it

Reply:
ok, I've set pre-commit up